src/components/producerItems/components/ProducerItems.js

```diff
@@ -130,18 +163,24 @@ class ProducerItems extends React.Component {
 
 ProducerItems.propTypes = {
     actions: PropTypes.shape({
+        requestProductDelete: PropTypes.func,
         requestProductList: PropTypes.func,
         requestProductUpdate: PropTypes.func,
     }),
+    errorMessage: PropTypes.string,
     productList: PropTypes.instanceOf(Map),
+    successMessage: PropTypes.string,
 };
 
 const mapStateToProps = state => ({
+    errorMessage: getErrorMessage(state),
     productList: getProductList(state),
+    successMessage: getSuccessMessage(state),
 });
 
 const mapDispatchToProps = dispatch => ({
     actions: bindActionCreators({
+        requestProductDelete,
         requestProductList,
         requestProductUpdate,
     }, dispatch),
```

Comment:
`bindActionCreators(actions, dispatch)`

Reply:
At this level, i followed the already method be done

Reply:
ok